src/ducks/balance/helpers.js

```diff
@@ -0,0 +1,59 @@
+import groupBy from 'lodash/groupBy'
+import sumBy from 'lodash/sumBy'
+import getEarliestDate from 'date-fns/min'
+import isDateAfter from 'date-fns/is_after'
+import isDateEqual from 'date-fns/is_equal'
+import subDays from 'date-fns/sub_days'
+import startOfToday from 'date-fns/start_of_today'
+import formatDate from 'date-fns/format'
+
+export const getBalanceHistories = (accounts, transactions) => {
+  if (accounts.length === 0 || transactions.length === 0) {
+    return null
+  }
+
+  const balances = accounts.reduce((balances, account) => {
+    balances[account.id] = getBalanceHistory(
+      account,
+      getTransactionsForAccount(account, transactions),
+      startOfToday()
+    )
+
+    return balances
+  }, {})
+
+  return balances
+}
+
+const getTransactionsForAccount = (account, transactions) =>
+  transactions.filter(t => t.account === account.id)
+
+export const getBalanceHistory = (account, transactions, from) => {
+  const transactionsByDate = groupBy(transactions, t =>
+    formatDate(t.date, 'YYYY-MM-DD')
```

Comment:
may be you can create a variable for this format `'YYYY-MM-DD'`

Reply:
:+1: